texts/formatted_evolution.txt

```diff
@@ -0,0 +1,1457 @@
+;	"come# to" vblex pp
+;	"come# to" vblex imp
+"<be>"
+	"be" vbser inf
+"<the>"
+	"the" det def sp
+"<way>"
+	"way" n sg
+"<they>"
+	"prpers" prn subj p3 mf pl
+"<are>"
+;	"be" vbser pres
+;	"be" vbser pres p1 pl
+;	"be" vbser pres p2 sg
+;	"be" vbser pres p2 pl
+	"be" vbser pres p3 pl
+
+"<Earth>"
+	"earth" n sg
+"<is>"
+	"be" vbser pres p3 sg
+"<very>"
+	"very" adv
+;	"very" preadv
+"<old>"
+	"old" adj sint
+"<.>"
+	"." sent
+"<By>"
+;	"by" adv
+	"by" pr
+"<studying>"
+;	"study" vblex pprs
+	"study" vblex ger
+;	"study" vblex subs
+"<the>"
+	"the" det def sp
+"<layers>"
+	"layer" n pl
+;	"layer" vblex pres p3 sg
+"<of>"
+	"of" pr
+"<rock>"
+	"rock" n sg
+;	"rock" vblex inf
+;	"rock" vblex pres
+;	"rock" vblex imp
+"<that>"
+	"that" cnjsub
+;	"that" det dem sg
+;	"that" prn dem mf sg
+;	"that" prn rel an mf sp
+"<make up>"
+	"make# up" vblex sep inf
+;	"make# up" vblex sep pres
+	"make# up" vblex sep imp
+"<Earth>"
+	"earth" n sg
+"<'s>"
+	"'s" gen
+	"be" vbser pres p3 sg
+	"have" vbhaver pres p3 sg
+"<crust>"
+	"crust" n sg
+"<,>"
+	"," cm
+"<scientists>"
+	"scientist" n pl
+"<can>"
+;	"can" n sg
+;	"can" vbmod inf
+	"can" vbmod pres
+;	"can" vblex inf
+;	"can" vblex pres
+;	"can" vblex imp
+"<find out>"
+	"find# out" vblex inf
+;	"find# out" vblex pres
+;	"find# out" vblex imp
+"<about>"
+;	"about" adv
+	"about" pr
+"<its>"
+;	"its" det pos sp
+	"prpers" det pos p3 nt sg
+"<past>"
+	"past" adj
+;	"past" pr
+;	"past" n sg
+"<.>"
+	"." sent
+"<This>"
+	"this" det dem sg
+;	"this" prn dem mf sg
+;	"this" prn rel an mf sp
+"<kind>"
+;	"kind" adj sint
+	"kind" n sg
+"<of>"
+	"of" pr
+"<research>"
+	"research" n sg
+;	"research" vblex inf
+;	"research" vblex pres
+;	"research" vblex imp
+"<is called>"
+	"be# called" vblex pres p3 sg
+"<historical>"
+	"historical" adj
+"<geology>"
+	"geology" n sg
+"<.>"
+	"." sent
+
+"<It>"
+	"prpers" prn subj p3 nt sg
+;	"prpers" prn obj p3 nt sg
+"<is>"
+	"be" vbser pres p3 sg
+"<known>"
+	"known" adj
+	"know" vblex pp
+"<that>"
+	"that" cnjsub
+;	"that" det dem sg
+;	"that" prn dem mf sg
+;	"that" prn rel an mf sp
+"<living>"
+	"live" vblex pprs
+;	"live" vblex ger
+;	"live" vblex subs
+;	"living" adj
+;	"living" n sg
+"<things>"
+	"thing" n pl
+"<have>"
+;	"have" vbhaver inf
+	"have" vbhaver pres
+;	"have" vblex inf
+;	"have" vblex pres
+;	"have" vblex imp
+"<changed>"
+;	"changed" adj
+	"change" vblex past
+;	"change" vblex pp
+"<over time>"
+	"over time" adv
+"<,>"
+	"," cm
+"<because>"
+	"because" cnjsub
+;	"because" cnjadv
+"<their>"
+	"their" det pos sp
+	"prpers" det pos p3 mf pl
+"<remains>"
+	"remains" n pl
+;	"remain" vblex pres p3 sg
+"<can>"
+;	"can" n sg
+;	"can" vbmod inf
+	"can" vbmod pres
+;	"can" vblex inf
+;	"can" vblex pres
+;	"can" vblex imp
+"<be>"
+	"be" vbser inf
+"<seen>"
+	"see" vblex pp
+"<in>"
+;	"in" adv
+	"in" pr
+"<the>"
+	"the" det def sp
+"<rocks>"
+	"rock" n pl
+;	"rock" vblex pres p3 sg
+"<.>"
+	"." sent
+"<These>"
+;	"this" det dem pl
+	"this" prn dem mf pl
```

Comment:
Determiner, not pronoun, I think.

Reply:
https://en.wikipedia.org/wiki/Demonstrative#Demonstrative_determiners_and_pronouns